spec/jobs/entity_table_check_job_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "rails_helper"
+
+RSpec.describe "EntityTableCheckJob" do
+  describe "#perform" do
+    it "runs the entity table check job" do
+      expect_any_instance_of(DfE::Analytics::EntityTableCheckJob).to receive(:perform)
+
+      DfE::Analytics::EntityTableCheckJob.new.perform
```

Comment:
This is calling the wrong class. It may be useful to make the naming more distinct.

Reply:
Now updated 👍